docs/sources/panels-visualizations/visualizations/datagrid/index.md

```diff
@@ -0,0 +1,117 @@
+---
+aliases:
+  - ../../features/panels/datagrid_panel/
+  - ../../reference/datagrid/
+  - ../../visualizations/datagrid/
+description: Learn about datagrid panel visualization features.
+keywords:
+  - grafana
+  - dashboard
+  - panels
+  - datagrid
+  - datagrid panel
+  - datagrid options
+menuTitle: Datagrid
+title: Datagrid
+weight: 350
+---
+
+# Datagrid
+
+{{% admonition type="note" %}}
+
+The Grafana Datagrid panel is experimental. This feature is supported by the engineering team on a best-effort basis, and breaking changes may occur without notice prior to general availability.
+
+{{% /admonition %}}
+
+The Datagrid panel offers you the ability to create, edit, and fine-tune data within Grafana. As such, this panel can act as a data source for other panels
+inside a dashboard.
+
+Through it, you can manipulate data queried from any data source, or from a dragged and dropped file. You can then use the panel as a simple tabular
+visualization, or you can modify the data—and even remove it altogether—to create a blank slate.
+
+Editing the dataset changes the data source to use the inbuilt `-- Grafana --` data source, thus replacing the old data source settings and related queries, while also copying the current dataset into the dashboard model. You can either use a pre-existing dataset from any other data source, start with a blank slate, or you can drag and drop a file to create a new dataset.
```

Comment:
I'll be changing the above first sentence to `Through it, you can manipulate data queried from any data source, you can start from a blank slate, or you can pull data from a dragged and dropped file.` so that it encapsulates the details and remove the last sentence of the second part alltogether.